### PR TITLE
[release] Update changelog for 0.2.32

### PR DESCRIPTION
**Pull request by tpoterba**

### CODE REVIEW

File: hail/python/hail/docs/change_log.md [hunk 1]
```diff
@@ -22,6 +22,32 @@ Please note that **forward compatibility should not be expected, especially
...
+
+### Critical performance regression fix
+
+- (hail#7989) Fixed performance regression leading to a large slowdown when `hl.variant_qc` was run after filtering columns.
+
+### Performance
+
+- (hail#7962) Improved performance of `hl.pc_relate`.
+- (hail#8032) Drastically improve performance of pipelines calling `hl.variant_qc` and `hl.sample_qc` iteratively.
+- (hail#8037) Improve performance of `hl.nd.array.dot` by using native linear algebra libraries.
```
Comment (johnc1231) [inline hail/python/hail/docs/change_log.md]: No such thing as `dot`. This is `NDArrayNumericExpression.__matmul__`

File: hail/python/hail/docs/change_log.md [hunk 2]
```diff
@@ -22,6 +22,32 @@ Please note that **forward compatibility should not be expected, especially
...
+
+- (hail#7989) Fixed performance regression leading to a large slowdown when `hl.variant_qc` was run after filtering columns.
+
+### Performance
+
+- (hail#7962) Improved performance of `hl.pc_relate`.
+- (hail#8032) Drastically improve performance of pipelines calling `hl.variant_qc` and `hl.sample_qc` iteratively.
+- (hail#8037) Improve performance of NDArray matrix multiply by using native linear algebra libraries.
+
+### Bug fixes
```
Comment (johnc1231) [inline hail/python/hail/docs/change_log.md]: Missing bugfixes:

- #8035
- #8021 
- #7986

File: hail/python/hail/docs/change_log.md [hunk 3]
```diff
@@ -22,6 +22,32 @@ Please note that **forward compatibility should not be expected, especially
...
+--- 
+
+## Version 0.2.32
+
+Released 2020-02-07
+
+### Critical performance regression fix
+
+- (hail#7989) Fixed performance regression leading to a large slowdown when `hl.variant_qc` was run after filtering columns.
+
```
Comment (johnc1231) [inline hail/python/hail/docs/change_log.md]: New functions:

- #8007

**Comment by tpoterba** [comment]
done, thanks!

**Pull request merged**